[plugin.video.newyankeeworkshop] v0.0.1

th0ma5w opened the pull request:
### Description
<!--- Provide a short summary of submitted add-on in case it's a new addition. -->
<!--- If it's plugin update only highlight biggest changes if needed. -->
<!--- Make sure you follow the checklist below before finalizing your pull-request. -->
This plugin allows for viewing of videos directly from the official website for The New Yankee Workshop at https://www.newyankee.com/ and will automatically update with new videos each month until the archive they are building is complete.
### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Lunatixz commented on plugin.video.newyankeeworkshop/newyankee.py:
```diff
@@ -0,0 +1,54 @@
+    for episode in episode_list:
+        episode_display, episode_link = episode
+        url = build_url({'mode': 'episode', 'season': season_link, 'episode': episode_link, 'display': episode_display})
+        li  = xbmcgui.ListItem(episode_display, iconImage="DefaultFolder.png")
```
iconImage, thumbnailImage are deprecated, please use setArt https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

Lunatixz commented on plugin.video.newyankeeworkshop/router.py:
```diff
@@ -0,0 +1,54 @@
+    for episode in episode_list:
+        episode_display, episode_link = episode
+        url = build_url({'mode': 'episode', 'season': season_link, 'episode': episode_link, 'display': episode_display})
+        li  = xbmcgui.ListItem(episode_display, iconImage="DefaultFolder.png")
```
`iconImage `is deprecated please use setArt.

th0ma5w commented:
Okay I believe I corrected all of the changes and rebased everything back to 1 commit. Feel free to let me know about any other possible changes needed or improvements and I'll be happy to update it. Thanks again!

Lunatixz commented:
LGTM, Thanks

Lunatixz commented:
@Rechi @enen92 needs second review

th0ma5w commented:
@Rechi @Lunatixz @basrieter I'd like to remove this plugin, and sorry for the delay on this. Is there a document I can follow? The original content creators have moved this context to their YouTube channel so this plugin is now not necessary with the YouTube plugin. Thank you so much for your time with this! Sorry for the ping.

basrieter commented:
Just submit a new (higher) version in which you   mark it as broken in the addon.xml. See https://kodi.wiki/view/Addon.xml#%3Cbroken%3E
